### ISSUE TITLE
HyInt and ETCCDI Trend and significance testing

### ISSUE THREAD

**Issue by yifatdzigan** [c3s-magic/c3s-magic-metric-status#17]
This is a tracking issue for the final integration of *HyInt and ETCCDI Trend and significance testing metric* into the MAGIC framework.

This is also a place for discussion and updates on the current status, problems and questions related to this metric.

**Links:**
----------

[Portal](https://portal.c3s-magic.eu/#/diagnostics/hyint_timeseries)
[Recipe](https://github.com/ESMValGroup/ESMValTool/blob/REFACTORING_hyint/esmvaltool/recipes/recipe_hyint.yml)
For an overview of the current status of all metrics see [MAGIC Metrics Final Integration Checklist](https://docs.google.com/spreadsheets/d/1LFhEuL6w9KBqFmujVXXN6qj98jigMIPArYkSVKB_Frw/edit#gid=0)

**Contacts:**
--------------
@jhardenberg
@earnone

**Comment by earnone** [comment]
The trend calculation and significance testing functionalities were included in the hyint main diagnostic (see #8) and as agreed there is no need anymore to keep these separate: The split of this part was only due to deliverables falling in two different workpackages (WP4 and WP6) of the magic project. The main diagnostic was merged into `version2_development`, see [https://github.com/c3s-magic/c3s-magic-metric-status/issues/17](https://github.com/c3s-magic/c3s-magic-metric-status/issues/17)

The inclusion of ETCCDI indices in hyint is already part of the code but was currently not called. It is being refined within the main hyint diagnostic in a separate branch in parallel with the new `extreme_events_v2` diagnostic. So this issue can be closed.

**Comment by yifatdzigan** [comment]
Still needs refinement of the outputs in the static diagnostic page

**Comment by earnone** [comment]
`Still needs refinement of the outputs in the static diagnostic page`

@yifatdzigan do you mean of the pre-calculated data shown with the viewer? Yes, all the names of the files/layers are too long, it would be enough to cut them short (maybe removing the long name of the model file). 

Or other parts (output=figures?)? If so can I see the current version somewhere?

**Comment by earnone** [comment]
>  If so can I see the current version somewhere?

I was not able to run the docker at this stage... sorry.
